src/swarm/neo/client/mixins/ClientCore.d

```diff
@@ -511,6 +621,31 @@ template ClientCore ( )
 
     /***************************************************************************
 
+        Writes stats about all requests to the provided stats log.
+
+        Params:
+            logger = stats log to write the filled instance of Aggr to
+
+    ***************************************************************************/
+
+    private void logRequestStats ( StatsLog logger )
```

Comment:
Although this is `private`, it will be accessible at the point where this template is mixed in, right?